lib/common/xml.c

```diff
@@ -18,7 +18,6 @@
 #include <sys/types.h>
 
 #include <glib.h>                       // gboolean, GString
-#include <libxml/parser.h>              // xmlCleanupParser()
```

Comment:
This wound up in the wrong commit but I'll leave it if there are no other issues